flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/consumer/SingleInputGateFactory.java

```diff
@@ -50,10 +50,11 @@
  * Factory for {@link SingleInputGate} to use in {@link NettyShuffleEnvironment}.
  */
 public class SingleInputGateFactory {
+	@SuppressWarnings("LoggerInitializedWithForeignClass")
 	private static final Logger LOG = LoggerFactory.getLogger(SingleInputGate.class);
```

Comment:
why not use `SingleInputGateFactory.class` here?

Reply:
My idea was to preserve the structure of the previous logging messages because the factory was a result of refactoring. I can change this if it is not important.

Reply:
got it, thanks for the explanation!